modules/dqt/ajax/saveQuery.php

```diff
@@ -81,6 +80,6 @@
     $query['id'] = $qid;
     print json_encode($query);
 } else {
-    print $cdb->postDoc($baseDocument);
+    print json_encode($cdb->postDoc($baseDocument));
```

Comment:
```
[Wed Jan 19 13:51:34.348020 2022] [php7:error] [pid 52387] [client 127.0.0.1:39804] PHP Fatal error:  Uncaught LorisException: Decoded JSON data returned null. Cannot continue. in /var/www/loris/php/libraries/CouchDB.class.inc:539\nStack trace:\n#0 /var/www/loris/modules/dqt/ajax/saveQuery.php(43): CouchDB->getDoc()\n#1 /var/www/loris/htdocs/AjaxHelper.php(112): require('/var/www/loris/...')\n#2 {main}\n  thrown in /var/www/loris/php/libraries/CouchDB.class.inc on line 539,
```
I get this

Reply:
Hi @ridz1208 I'm not getting this error when saving democraphics. Did you do anything special to trigger this bug?

Reply:
@maltheism it seems to be related to spaces in the query name

Reply:
Hi @ridz1208, I fixed that bug with the latest commit.